web/src/pm_conversations.ts

```diff
@@ -19,8 +19,8 @@ export function is_partner(user_id: number): boolean {
 }
 
 function filter_muted_pms(conversation: PMConversation): boolean {
-    // We hide muted users from the top left corner, as well as those huddles
-    // in which all participants are muted.
+    // We hide muted users from the top left corner, as well as those direct
+    // message groups in which all participants are muted.
     const recipients = people.split_to_ints(conversation.user_ids_string);
 
     if (recipients.every((id) => muted_users.is_user_muted(id))) {
```

Comment:
Maybe a good follow-up project would be renaming this module to `direct_messages.ts` while we're working on this sweep? Probably worth posting a proposal in #frontend with suggested names for each of the functions that have `pm` or `private` in them before doing the work, since 90% of review will be bikeshedding the names.

Reply:
sure!